config/application.rb

```diff
@@ -1,21 +1,29 @@
-require_relative 'boot'
+require_relative "boot"
 
 require "rails"
+# Pick the frameworks you want:
 require "active_model/railtie"
+require "active_job/railtie"
```

Comment:
This application does not use Active Job.

Reply:
Fixed